tests/test_extend_schema.yml

```diff
@@ -206,6 +206,86 @@ paths:
       responses:
         '204':
           description: No response body
+  /doesitall/post_with_example/:
```

Comment:
same here i guess. since this is now part of `test_examples.py` if can be safely removed as it should be covered there completely